Makefile

```diff
@@ -18,12 +18,12 @@ help:
 # Turn them all off. We'll turn them back on to try to get to working tests.
 MAINBOARDS := $(wildcard src/mainboard/*/*/Makefile)
 
-TOOLCHAIN_VER := $(shell grep channel rust-toolchain.toml | grep -e '".*"' -o)
 BINUTILS_VER := 0.3.4
 STACK_SIZES_VER := 0.4.0
 TARPAULIN_VER := 0.19.1
+DPRINT_VER := 0.32.2
 
-CARGOINST := rustup run --install nightly cargo install
+CARGOINST := rustup run --install 1.65 cargo install
```

Comment:
stable?

Reply:
I am pinning the Rust release (from today :)) and we also pinned each of the tools to a specific version - that gets us consistency. The previous "just use stable" was a pragmatic choice, I think because something didn't build otherwise back then.

Reply:
Makes sense. Thanks.